drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4977,6 +4980,15 @@ void RasterizerSceneGLES3::initialize() {
 		storage->material_set_shader(default_overdraw_material, default_overdraw_shader);
 	}
 
+	{
+		//default material and shader for light affect debug
+
+		default_light_affect_debug_shader = storage->shader_create();
+		storage->shader_set_code(default_light_affect_debug_shader, "shader_type spatial;\nrender_mode ambient_light_disabled,blend_add;\n void light() { SPECULAR_LIGHT.r+=1.0; int lc=int((SPECULAR_LIGHT.r)+0.1); if(lc<=4) { DIFFUSE_LIGHT=vec3(0.1, 0.3, 0.5); } else if(lc<=6) { DIFFUSE_LIGHT=vec3(0.3, 0.1, 0.7); } else if(lc<=7) { DIFFUSE_LIGHT=vec3(0.8, 0.6, 0.1); } else if(lc>=8) { DIFFUSE_LIGHT=vec3(1.0, 0.1, 0.1); } }\n void fragment() { SPECULAR=0.0; ALBEDO=vec3(1.0); EMISSION=vec3(0.1,0.1,0.1); ALPHA=0.4; }");
+		default_light_affect_debug_material=storage->material_create();
```

Comment:
Style issue here, there should be spaces around `=`. See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Oops, missed that somehow, it wasn't intentional.